db/db_impl_write.cc

```diff
@@ -389,6 +389,8 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
   }
   if (should_exit_batch_group) {
     if (status.ok()) {
+      // Note: if we are to resume after non-OK statuses we need to revisit how
+      // we reacts to non-OK statuses here.
```

Comment:
I think we should mention the problem is related to PreReleaseCallback(). Otherwise people might not understand what it means.

Reply:
To be honest I do not fully understand the problem either. Can you feed me the line that I should paste in the code?